src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -4040,10 +4040,11 @@ public void setMinimumFractionDigits(int newValue) {
     }
 
     /**
-     * Gets the maximum number of digits allowed in the integer portion of a
-     * number. The maximum number of integer digits can be set by either {@link #setMaximumIntegerDigits(int)}
-     * or {@link #applyPattern(String)}. See the {@link ##patterns Pattern Section} for
-     * comprehensive rules regarding maximum integer digits in patterns.
+     * {@inheritDoc}
+     * <p>The maximum number of integer digits can be set by either {@link
+     * #setMaximumIntegerDigits(int)} or {@link #applyPattern(String)}.
+     * See the {@link ##patterns Pattern Section} for comprehensive rules regarding
+     * maximum integer digits in patterns.
```

Comment:
This part only exists in this method and other getter methods do not have one. Either removing this (as this is explained in the class description), or add it to all other getters would be consistent ( the former is better IMO).

Reply:
That commentary is only in this method because of the non-obvious behavior for the maximum integer digits. (The pattern does not change the `maximumIntegerDigits`). Since not all users may read that section in the class description, that is why this method has the explicit call-out, so I think it may be beneficial to keep this wording. I am fine either way, if you don't think it's worth it, I'll remove.

Reply:
If there is some oddity in this getter, would it be helpful if we add more to it, eg, "Unlike other getter methods..."?

Reply:
That's a good point, it should be phrased as a call out the unique behavior. New wording is also more concise. Changed in https://github.com/openjdk/jdk/pull/24265/commits/5fc8f37f951555a48b78a133a528e298faabd9b7.